eq-author/src/App/QuestionnairesPage/QuestionnairesTable/index.js

```diff
@@ -107,30 +134,18 @@ export class UnconnectedQuestionnairesTable extends React.PureComponent {
     }
 
     return (
-      <>
+      <Panel>
         <Table>
           <thead>
             <tr>
-              <TH colWidth="50%">Questionnaire name</TH>
-              <TH colWidth="15%">Date</TH>
-              <TH colWidth="22%">Created by</TH>
-              <TH colWidth="14%" />
+              <TH colWidth="45%">Title</TH>
```

Comment:
These percentages don't add up to 100

Reply:
Worth writing a test for this?

Reply:
I'm not sure how much it matters as it only affects layout like CSS which we don't test directly. Its just odd.

Reply:
Changed it so it adds up to 100, however, im not sure what they should be, just picked something that looked good.